packages/bvaughn-architecture-demo/src/suspense/MappedLocationCache.ts

```diff
@@ -5,10 +5,11 @@ import {
 import { ReplayClientInterface } from "shared/client/types";
 import { createGenericCache } from "./createGenericCache";
 
-export const { getValue: getMappedLocation } = createGenericCache<
-  [ReplayClientInterface, ProtocolLocation],
-  ProtocolMappedLocation
->(
+export const {
+  getValueSuspense: getMappedLocationSuspense,
```

Comment:
:speech_balloon: huh, this is kinda similar to RTK's `createEntityAdapter` returning several premade selectors. Neat!